pkg/rpcclient/wsclient_test.go

```diff
@@ -276,6 +276,7 @@ func TestWSClientEvents(t *testing.T) {
 	require.False(t, ok)
 	_, ok = <-ntfCh
 	require.False(t, ok)
+	t.Cleanup(srv.Close)
```

Comment:
Should be placed right after `srv` construction because if test breaks in the middle of execution, the server won't be properly closed.